package.json

```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
-    "hof-bootstrap": "^6.0.0",
-    "hof-frontend-toolkit": "^1.0.0",
+    "express": "^4.14.0",
```

Comment:
That's what I did initially but my linting complained
```mock-postcode.js
  2:16  error  Module "express" is not listed as a dependency in package.json  implicit-dependencies/no-implicit```

Does this mean the linting is broken? Should I fix the linting? In the mean time should I put in a `es-lint ignore line`